### ISSUE TITLE
Split the python script invocation to pattern selection and execution

### ISSUE THREAD

**Issue by radomirbosak** [radomirbosak/shellcut#19]
`shellcut` should get two new CLI argument/switches, so that:
```
./shellcut.py --prompt "https://github.com/google/python-fire/blob/master/README.md"
```
would prompt user if the input data (a url in this case) matches multiple patterns, while
```
./shellcut.py --choose=1 "https://github.com/google/python-fire/blob/master/README.md"
```
would choose the first matching pattern from those available, and the only script output will be the bash/fish script to execute

**Comment by radomirbosak** [comment]
The reason for implementing this is that currently, everything outputed by `shellcut.py` is interpreted by the shell. This means that it cannot interact with user properly.

**Comment by radomirbosak** [comment]
This should be implemented if #15 is not.

**Comment by radomirbosak** [comment]
Since #15 was implemented, I am closing this one
